python_translators/translation_response.py

```diff
@@ -0,0 +1,7 @@
+from python_translators.translation_costs import TranslationCosts
+
+
+class TranslationResponse(object):
+    def __init__(self, translations: [str], costs: TranslationCosts):
```

Comment:
one more thing for the translation response: it would be cool to add also the source. so when we display it to the user, we could in theory show them how the translation has been obtained.